modules/appeals_api/config/schemas/v2/10182_headers.json

```diff
@@ -26,6 +26,12 @@
         "X-Consumer-Username":  { "$ref": "#/definitions/X-Consumer-Username"},
         "X-Consumer-ID":        { "$ref": "#/definitions/X-Consumer-ID"}
       },
+      "dependencies": {
```

Comment:
Almost forgot. I reduced the amount of repetition in the list of dependencies (see contrast with current hlr v2 claimant dependencies).  With each attribute requiring all the rest, as it still is in hlr v2, if two attributes are missed, the error messages are cumbersome and repetitive. For example, on hlr v2 if you don't add claimant first name, but you add all the other attributes, it will repeat the 'x va claimant first name required' error/pointer 3 times. I hope this makes sense, and I think we should improve it on hlr and sc if necessary :)